modules/webbrowser/include/modules/webbrowser/interaction/cefinteractionhandler.h

```diff
@@ -67,14 +67,12 @@ class IVW_MODULE_WEBBROWSER_API CEFInteractionHandler : public InteractionHandle
      * host must be provided before invokeEvent is called.
      * @param host The browser to inject events into.
      */
-    CEFInteractionHandler(CefRefPtr<CefBrowserHost> host = nullptr);
+    explicit CEFInteractionHandler(CefRefPtr<CefBrowserHost> host = nullptr);
     virtual ~CEFInteractionHandler() = default;
 
-    virtual std::string getClassIdentifier() const override {
-        return "org.inviwo.cefinteractionhandler";
-    }
+    std::string getClassIdentifier() const override { return "org.inviwo.cefinteractionhandler"; }
 
-    virtual void invokeEvent(Event* event) override;
+    void invokeEvent(Event* event) override;
```

Comment:
In inviwo we have kept both virtual and override. Mostly for legacy and old habits. I suppose, But if we decide on changing that we should try and do it in a coordinated way.

Reply:
I.e. don't remove them now

Reply:
Created issue #1569 on that.